### PR TITLE
Split user manual in chapter files

### PR DESCRIPTION
**Pull request by hiltonlima**
https://github.com/project-chip/certification-tool/blob/split_user_guide_file/docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

### CODE REVIEW

**Review by rquidute — COMMENTED** [review]
Could you please update the year in the copyright text to `2024`?

**Review by rquidute — APPROVED** [review]

File: docs/Matter_TH_User_Guide/11-TestCaseExecution.adoc [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+////
+<<<
+== *Test Case Execution*
+
+Refer to <<references, Section 2, References>> for PICS tool documentation to generate the PICS XML files.
```
Comment (antonio-amjr) [inline docs/Matter_TH_User_Guide/11-TestCaseExecution.adoc:20]: Is `References` still section 2?

File: docs/Matter_TH_User_Guide/3-TableOfContents.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+////
+<<<
+[discrete]
+== *Table of contents*
+toc::[]
```
Comment (antonio-amjr) [inline docs/Matter_TH_User_Guide/3-TableOfContents.adoc:20]: Table of content is not showing nothing

File: docs/Matter_TH_User_Guide/11-TestCaseExecution.adoc (hunk 1 above)
Comment (antonio-amjr) [reply docs/Matter_TH_User_Guide/11-TestCaseExecution.adoc:20]: Review all other asciidoc references as well

**Review by antonio-amjr — CHANGES_REQUESTED** [review]
Take a look in the comments above and solve the conflicts of this PR

File: docs/Matter_TH_User_Guide/11-TestCaseExecution.adoc (hunk 1 above)
Comment (antonio-amjr) [reply docs/Matter_TH_User_Guide/11-TestCaseExecution.adoc]: The reference seems ok for the whole document. No issue there

File: docs/Matter_TH_User_Guide/3-TableOfContents.adoc (hunk 2 above)
Comment (antonio-amjr) [reply docs/Matter_TH_User_Guide/3-TableOfContents.adoc]: The Table of Content shows the information for the whole document.

**Review by antonio-amjr — APPROVED** [review]

**Pull request merged**